src/tests/ftest/pool/destroy_rebuild.py

```diff
@@ -50,7 +50,7 @@ def test_destroy_while_rebuilding(self):
             "Invalid pool information detected prior to rebuild")
 
         # Start rebuild
-        self.pool.start_rebuild([rank], self.d_log)
+        self.server_managers[0].stop_ranks([rank], self.d_log)
         self.pool.wait_for_rebuild(True)
```

Comment:
Question: We don't need `pool_query_timeout: 30` in it's yaml file? I understand we are just waiting for rebuild to start and not for it to complete, is that the reason?

Reply:
Good catch.  When the `@skipForTicket("DAOS-2723")` is removed it will most likely need `pool_query_timeout: 30` in it's yaml file

Reply:
I'll take care of this in the follow on PR where we'll be debugging the pool/rebuild_test.py issues.